src/components/Slides.vue

```diff
@@ -141,9 +144,28 @@ export default {
       ]
     }
   },
-  methods: {
-    searchForSlide () {
-      console.log('Pressed search')
+  computed: {
+    filteredSlides () {
+      // check if something is typed into the search bar
+      if (this.searchString) {
+        // if there is something the in the search bar then filter the array fo current slides
+        let vm = this
+        // search the list of slides
+        let fSlides = this.slides.filter(
```

Comment:
`fSlides` would leek better if it was called `filtered_slides`
... I'm assuming the 'f' stands for filtered?

Reply:
I'll update that

Reply:
ESLint wants me to use camel case, so I'll use filteredSlides

Reply:
gross' @KingMackle whyyyyyyy

Reply:
everything else is using camel case